### ISSUE TITLE
Socket Close Abnormally 1006

### ISSUE THREAD

**Issue by sonawanesarvesh** [johnnymast/mysql_websocket_chat#47]
Hello, 
I followed your deployment instruction on one web server, but I am getting this error

WebSocket connection to 'wss://<myIP>:8080/' failed:
We have received an error! Event {isTrusted: true, type: "error", target: WebSocket, currentTarget: WebSocket, eventPhase: 2, …}
Connection closed  with event code 1006 => The connection was closed abnormally, e.g., without sending or receiving a Close control frame

**Comment by johnnymast** [comment]
Hi there,

I am sorry for my late response. Are you running the server script in the background?

**Comment by LP233** [comment]
I'm also getting same error
![terminal](https://user-images.githubusercontent.com/26641741/115506145-28e8b480-a298-11eb-9af8-84638a0a393a.png)
![front](https://user-images.githubusercontent.com/26641741/115506152-2a19e180-a298-11eb-8a48-2d5012981e65.png)

**Comment by johnnymast** [comment]
Allright boys, let me check it out.

**Comment by johnnymast** [comment]
Hi guys,

I see that you guys are using wss:// (secure) but the 1.4 version does not support SSL (yet). The [upcoming milestone (1.5 Add more security to the chats)](https://github.com/johnnymast/mysql_websocket_chat/milestone/5) will feature SSL and docker support for the project. You can follow the link to track the progress.

**Comment by sonawanesarvesh** [comment]
Hello, thanks for the updates, but I see no latest comments on https://github.com/johnnymast/mysql_websocket_chat/issues/8

is it stalled since long? is there any hopes to see updates in some days (not in weeks :) )

**Comment by johnnymast** [comment]
That's right it's a ticket I added for the v1.5 milestone. I have 98 repo's to maintain so I didn't come to it (yet) but since yesterday I have been working on the 1.5 branch again. If you wish to help, all help is welcome :)

**Comment by sonawanesarvesh** [comment]
Hello 
my team tried to work on
https://stackoverflow.com/questions/53989596/unable-to-connect-to-websocket-using-wss-in-php
https://www.php.net/manual/en/class.streamwrapper.php

Just wanted confirm the right direction?

**Comment by johnnymast** [comment]
Hi sonawanesarvesh,

I can't really tell because mysql_websocket_chat uses a third-party WebSocket server and I don't know what it does in its core. But for version 1.5 I am adding certs creator to the package.  

https://github.com/johnnymast/mysql_websocket_chat/blob/v1.5/ssl/cert.php

Maybe if you have certs you know are 100% valid you could help me test version 1.5 before release. You can do so by updating this line in the server https://github.com/johnnymast/mysql_websocket_chat/blob/v1.5/server.php#L9 and enable SSL support here https://github.com/johnnymast/mysql_websocket_chat/blob/v1.5/includes/config.php#L25 (Don't forget to download the 1.5 branch the other branches/versions don't have this option.

**Comment by johnnymast** [comment]
@sonawanesarvesh I have the solution but I can not work with the version of mysql_websocket_chat you work with now. Really soon I will release 1.5 pre-release with SSL support. I had to debug Ratched WebSockets head to toe to see why it doesn't like SSL in 1.5. This took a while but I got it done now. Expect a pre-release + youtube video soon to fix this.
